From: David Steele <steele@debian.org>
Date: Sun, 17 Sep 2017 20:24:28 -0400
Subject: Don't make comitup.json part of the package

We don't want it to be overwritten on upgrade.

--- a/setup.py
+++ b/setup.py
@@ -47,7 +47,6 @@
     },
     data_files=[
         ("/etc", ["conf/comitup.conf"]),
-        ("/var/lib/comitup", ["conf/comitup.json"]),
         ("/usr/share/dbus-1/system.d", ["conf/comitup-dbus.conf"]),
         (
             "/usr/share/comitup/web/templates",
