--- a/setup.py
+++ b/setup.py
@@ -49,7 +49,7 @@
         ("/etc", ["conf/comitup.conf"]),
         ("/usr/share/dbus-1/system.d", ["conf/comitup-dbus.conf"]),
         (
-            "/usr/share/comitup/web/templates",
+            "/web/templates",
             [
                 "web/templates/index.html",
                 "web/templates/connect.html",
@@ -57,7 +57,7 @@
             ],
         ),
         (
-            "/usr/share/comitup/web/templates/css",
+            "/web/templates/css",
             [
                 "web/templates/css/uikit.css",
                 "web/templates/css/uikit-rtl.css",
@@ -66,7 +66,7 @@
             ],
         ),
         (
-            "/usr/share/comitup/web/templates/js",
+            "/web/templates/js",
             [
                 "web/templates/js/uikit",
                 "web/templates/js/uikit-icons",
@@ -74,14 +74,14 @@
             ],
         ),
         (
-            "/usr/share/comitup/web/templates/images",
+            "/web/templates/images",
             [
                 "web/templates/images/ledon.gif",
                 "web/templates/images/ledoff.gif",
             ],
         ),
         (
-            "/usr/share/comitup/dns",
+            "/dns",
             [
                 "conf/dns-hotspot.conf",
                 "conf/dns-connected.conf",
